sections/03_categorize.md

```diff
@@ -143,21 +143,26 @@ This indicates a potential strength of deep methods. It may be possible to
 repurpose features from task to task, improving overall predictions as the field
 tackles new challenges.
 
-Several authors have created reusable feature sets for medical terminologies using
-neural embeddings, as popularized by word2Vec [@tag:Word2Vec]. This approach
-was first used on free text medical notes by De Vine et al.
-[@doi:10.1145/2661829.2661974] with results at or better than traditional methods.
-Y. Choi et al.[@doi:10.1145/2567948.2577348] built embeddings of standardized
-terminologies, such as ICD and NDC, used in widely available administrative
-claims data. By learning terminologies for different entities in the same
-vector space, they can potentially find relationships between different
-domains (e.g. drugs and the diseases they treat). Medical claims data does not
-have the natural document structure of clinical notes, and this issue was
-addressed by E. Choi et al. [@doi:10.1145/2939672.2939823], who built
-embeddings using a multi-layer network architecture which mimics the structure
-of claims data. While promising, difficulties in evaluating the quality of
-these kinds of features and variations in clinical coding practices remain as
-challenges to using them in practice.
+Several authors have created reusable feature sets for medical terminologies
+using natural language processing (NLP) and neural embedding models, as
+popularized by Word2vec [@tag:Word2Vec]. A goal of learning terminologies
+for different entities in the same vector spaec is to find relationships
+between different domains (e.g. drugs and the diseases they treat). It is
+difficult for us to provide a strong statement on the broad utility of these
+methods. Manuscripts in this area tend to compare algorithms applied to the
+same data but lack a comparison against overall best-practices for one or more
+tasks addressed by these methods. Techniques have been developed for free text
+medical notes [@doi:10.1145/2661829.2661974], ICD and NDC, and claims data
+[@doi:10.1145/2939672.2939823]. Methods for neural embeddings learned from
+electronic health records have at least some ability to predict disease-disease
+associations [@doi:10.1038/srep32404]. However, the evaluation did not
+differentiate between simple predictions (i.e. the same disease in different
+sites of the body) and non-intutive ones. While promising, a lack of rigorous
+evaluations of the real-world utility of these kinds of features makes
+current contributions in this area difficult to evaluate. To examine the true
+utility, comparisons need to be performed against leading approaches (i.e.
+algorithms and data) as opposed to simply evaluating multiple algorithms on
+the same potentially limited dataset.
```

Comment:
@cgreene, I think your summary looks good - my only concern is that the second model (DAG2D) in #342 is not even mentioned anymore. To me, that is the most interesting part of the paper: being able to identify disease-associated genes from EHR data has big implications for drug discovery (and specifically for target identification). It might be that DAG2D suffers from the same flaws of D2D but my preference would have been to mention this particular application (which doesn't seem to be explored elsewhere).
CC @brettbj @agitter